lib/swig/GeNNModel.py

```diff
@@ -15,14 +55,14 @@ class GeNNModel( object ):
     """
 
     def __init__(self, precision=None, modelName='GeNNModel', enableDebug=False,
-                 autoInitSparseVars=False, cpuOnly=False):
+                 autoInitSparseVars=True, cpuOnly=False):
```

Comment:
I don't think there's even any point exposing this at all - the behaviour when ``autoInitSpareVars=False`` is going to be removed in the next major version.